include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -100,6 +100,18 @@ class MGConstrainedDoFs : public Subscriptor
     const std::set<types::boundary_id> &boundary_ids,
     const ComponentMask &               component_mask = ComponentMask());
 
+  /**
+   * Fill the internal data structures with information
+   * about no normal flux boundary dofs.
+   *
+   * This function only works for a mesh whose no normal flux boundaries
+   * have faces which are normal to the x, y, or z axis.
```

Comment:
mention that we only support colorized meshes for now (copy text from the assert)